src/components/results/candidate-results/personality-archetype/index.js

```diff
@@ -33,24 +34,50 @@ class PersonalityArchetype extends Component {
     const personality = this.props.assessment.archetype;
     if(!personality) { return null; }
     const badge = personality.details.find(({title}) => title === "Badge");
-    const description = personality.details.find(({title}) => title === "Candidate Description");
+    const perspective = this.props.getOption("perspective");
+    const description = (
+      perspective === "thirdPerson"
+        ? personality.details.find(({title}) => title === "Hiring Manager Description")
+        : personality.details.find(({title}) => title === "Candidate Description")
+    );
 
     return (
       <div className={style.container}>
-        <div className={style.details}>
-          {badge && <img alt={personality.name} src={badge.body} />}
-          <DangerousHTML
-            html={this.props.translate("personality_heading", {
-              deck_name: this.props.deck.name,
-              personality: `<span>${personality.name}</span>`
-            })}
-            tag="h2"
-          />
-          {description && <p>{description.body}</p>}
-        </div>
-        <div className={style.meaning}>
-          <DangerousHTML html={this.props.translate("candidate_description_for_archetype_html")} tag="p" />
-        </div>
+        {perspective === "thirdPerson" ? (
+          <div className={style.thirdPersonDetails}>
+            <div className={style.itemOne}>
+              {badge && <img alt={personality.name} src={badge.body} />}
+              <div className={style.deckAndHeading}>
+                <DangerousHTML
+                  html={this.props.translate("personality_heading", {
+                    deck_name: this.props.deck.name,
+                    personality: personality.name
+                  })}
+                  className={style.personalityHeading}
+                  tag="h2"
+                />
+                <h2><span>{personality.name}</span></h2>
```

Comment:
Now it uses personality name twice (here and above). I think to try and handle what you're doing, we could do something like:

```
personality: `<br /><span>${personality.name}</span><br />`
```

That way the personality name will be on it's own line no matter what locale

Reply:
Hope it's ok that I ended up fixing this. Ran into a weird issue with Japanese while testing it out that I wanted to push up a fix for

Reply:
I don't mind at all, thanks for the help. I added your commit changes to my last commit because I hadn't pulled your changes down prior to my updates. Do you know a better git workflow for handling this case where I didn't pull your changes before making a commit? I ended up added your changes and force pushing my commit.

Reply:
I just `git pull origin master` and it puts the unpushed local commits or work on top. My default git pull method is rebase, so you might have to add a flag for that if it's not yours

Reply:
Or `git pull` whatever branch, didn't meant master there

Reply:
I added a few tests to bump our coverage back up, so you can try it again :)

Reply:
Both worked great :)